impl_core/src/test/java/io/opencensus/implcore/trace/export/SampledSpanStoreImplTest.java

```diff
@@ -171,19 +171,20 @@ public void addSpansWithoutRegisteredNamesInAllLatencyBuckets() {
   }
 
   @Test
-  public void registerAndUnregisterSpanNames() {
-    addSpanNameToAllLatencyBuckets(NOT_REGISTERED_SPAN_NAME);
+  public void registerUnregisterAndListSpanNames() {
+    assertThat(
+        sampleStore.listRegisteredSpanNamesForCollection().contains(REGISTERED_SPAN_NAME))
+        .isTrue();
     assertThat(
-            sampleStore.getSummary().getPerSpanNameSummary().containsKey(NOT_REGISTERED_SPAN_NAME))
+        sampleStore.listRegisteredSpanNamesForCollection().contains(NOT_REGISTERED_SPAN_NAME))
```

Comment:
I think it would be better to check the exact contents of the set with `containsExactly`, here and below.

Reply:
Done.